src/main/resources/model/instance.json

```diff
@@ -257,6 +257,15 @@
               "index": "multilang"
             }
           }
+        },
+        "callNumberPrefix": {
+          "index": "source"
```

Comment:
Why these values included in source? From my viewpoint, they won't be returned to UI in general use

Reply:
Removed them, we can add them again when needed.